src/router/guards/data.js

```diff
@@ -26,8 +28,12 @@ export default async function dataGuard() {
     // 2) Load organizations of the current user
     await loadUserOrganizations();
 
-    // 3) Load spaces and projects of the current user
-    await Promise.all([loadUserSpaces(), loadUserProjects()]);
+    // 3) Load spaces, projects and invitations of the current user
+    await Promise.all([
+      loadUserSpaces(),
+      loadUserProjects(),
+      loadUserInvitations()
+    ]);
```

Comment:
I don't think the comments are usefull here... The functions are well named, and you await them so yes it is important to await them before calling the following. The comments are juste describing what the code already tells you :)

Reply:
The code i commented is not from this PR... my bad